app/components/language_picker_component.html.erb

```diff
@@ -11,6 +11,7 @@
     <span id="language-picker-description-<%= unique_id %>" class="language-picker__label-text">
       <%= t('i18n.language') %>
     </span>
+    <%= render IconComponent.new(icon: :expand_more, size: 3, class: 'language-picker__expander') %>
```

Comment:
The specific use of the `expand_more` was intentional to enable reuse / caching with the same icon used in the banner for "Here's how you know".

Reply:
That's an interesting thought, though I think it'd be tricky to implement in practice, since both load the image indirectly through CSS. I've been curious to use a bit more of the [Chrome DevTools protocol](https://chromedevtools.github.io/devtools-protocol/) like we did with [browser emulation](https://github.com/18F/identity-idp/blob/main/spec/support/features/browser_emulation_helper.rb) and that might be an option here (e.g. monitoring [Network](https://chromedevtools.github.io/devtools-protocol/tot/Network/)], but even with that could be hard to pinpoint what loads what.